src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/nopol/AbstractNopolRepair.java

```diff
@@ -86,6 +88,14 @@ public void initWithJobStatus() {
         this.setSources(jobStatus.getRepairSourceDir());
     }
 
+    public void checkToolsJar() throws ClassNotFoundException {
+    	// Check if tools.jar is in classpath, as Nopol depends on it.
+        // This is not done in compile time as we don't want Repairnator to it to compile on OpenJDK or JDK >= 9
+        URLClassLoader loader;
```

Comment:
duplicate code with NopolMojo? Reuse the same method?

Reply:
I didn't do this initially because they are in different modules. Should I move the checkToolsJar() method to repairnator-core then?